docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -23,11 +23,12 @@ The main features of the Image Util API include:
 
 - Decoding from a file or memory and encoding to a file or memory
 
-  You can [decode images](#decode) and [encode them](#encode) with the following formats:
+  You can [decode images](#decode), [encode them](#encode) and [encode animation](#animation) with the following formats:
 
   - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
-  - Input image formats for decoding: JPEG, PNG, GIF, and BMP
-  - Output image formats for encoding: JPEG, PNG, GIF, BMP, and [animated GIF](#animation)
+  - Input image formats for decoding: JPEG, PNG, GIF, BMP and WEBP
+  - Output image formats for encoding: JPEG, PNG, GIF, BMP and WEBP
+  - Output image formats for encoding animation: [GIF and WEBP](#animation)
 
   > **Note**
```

Comment:
Global: Please ensure to use the latest note style all over the file. 

> [!NOTE]
>  your text ....

Reply:
Thank you for your comment, I updated the latest note style all over the file.